server/aesdsocket.c

```diff
@@ -650,4 +589,4 @@ int main(int argc, char* argv[])
 close_graceful();
 return 0;
 
-}
\ No newline at end of file
+}
```

Comment:
@shni9045 add the `.gch` file below to .gitignore, you shouldn't need this in your repo.